packages/spreadsheet-view/src/SpreadsheetView/components/Spreadsheet.js

```diff
@@ -195,13 +198,9 @@ export default pluginManager => {
     if (sortSpec) {
       const { descending } = sortSpec
       return (
-        <Icon
-          fontSize="small"
-          className={classes.sortIndicator}
-          onClick={sortSpec.switchDirection}
-        >
-          {descending ? 'keyboard_arrow_up' : 'keyboard_arrow_down'}
-        </Icon>
+        <div fontSize="small" className={classes.sortIndicator}>
+          {descending ? <ArrowUp /> : <ArrowDown />}
+        </div>
```

Comment:
fontSize and className on the containing div doesn't seem to be working
```suggestion
        descending ? (
          <KeyboardArrowUpIcon
            fontSize="small"
            className={classes.sortIndicator}
          />
        ) : (
          <KeyboardArrowDownIcon
            fontSize="small"
            className={classes.sortIndicator}
          />
        )
```